tests/logger/test_log_entry.py

```diff
@@ -326,6 +332,9 @@ def test_serialization(self, train_remove_log_entry_as_dict):
 
             assert train_remove_log_entry.to_dict() == {
                 "id": str(train_remove_log_entry.id),
+                # pylint: disable=no-member
```

Comment:
Just from curiosity, why do you have to disable the linter? 🤔

Reply:
We are using DateTimeField which corresponds with a DateTime object, but the linter does not recognize this. I think we had the same problem last week. If you have an idea how to solve this, I am open, but did not find anything

Reply:
Thanks! 💚